### PR TITLE
Fixed broken links & remove link to unknown page

### PR DESCRIPTION
**Pull request by robindirksen1**
There was 1 link to a non-existing page.. so I think that it should be removed until there is a specific page for that feature in the docs.

### CODE REVIEW

File: docs/general/errors.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Working with errors
 ---
 
-Whenever an error gets reported by your application, we show it on the projects page. If you want to know more about the error, click on it to see [more details about the last occurrence](/docs/general/working-with-error-occurrences).
+Whenever an error gets reported by your application, we show it on the projects page. If you want to know more about the error, click on it to see [more details about the last occurrence](/docs/general/sharing-error-occurrences).
```
Comment (riasvdv) [inline docs/general/errors.md]: That's not the right link, this should be /docs/general/error-occurrences

File: docs/ignition-for-laravel/installation.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ This will publish two files: `config/ignition.php` and `config/flare.php`.
 
 ## Sending errors to Flare
 
-If you want to send errors to Flare, you must specify a valid API key in the `key` variable of the config file or as your `FLARE_KEY` environment variable. You can get an API key when [creating a new project](/docs/general/projects) or by visiting the [project settings screen](/docs/general/configuring-projects) for an existing project.
```
Comment (riasvdv) [inline docs/ignition-for-laravel/installation.md]: Keep the text but remove the broken link for now

Comment (robindirksen1) [reply docs/ignition-for-laravel/installation.md]: Fixed

File: docs/general/errors.md (hunk 1 above)
Comment (robindirksen1) [reply docs/general/errors.md]: Fixed

**Comment by riasvdv** [comment]
Thanks!

**Pull request merged**